From 8a86359b7a814c352c729a63407f604d6241e19f Mon Sep 17 00:00:00 2001
From: nailcui <nailcui@foxmail.com>
Date: Tue, 6 Feb 2024 15:47:20 +0800
Subject: [PATCH] ZOOKEEPER-4770: zkSnapshotRecursiveSummaryToolkit.sh Error:
 Could not find or load main class

Reviewers: tisonkun, eolivelli, maoling
Author: nailcui
Closes #2091 from nailcui/ZOOKEEPER-4770

Upstream-commit: https://github.com/apache/zookeeper/commit/8a86359b7a814c352c729a63407f604d6241e19f
Upstream-issue: https://issues.apache.org/jira/browse/ZOOKEEPER-4770 (fixVersion 3.10.0, not in any 3.9.x release)
---
 bin/zkSnapshotRecursiveSummaryToolkit.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bin/zkSnapshotRecursiveSummaryToolkit.sh b/bin/zkSnapshotRecursiveSummaryToolkit.sh
index f7659971b1b..888fdc01138 100755
--- a/bin/zkSnapshotRecursiveSummaryToolkit.sh
+++ b/bin/zkSnapshotRecursiveSummaryToolkit.sh
@@ -33,7 +33,7 @@ else
   . "$ZOOBINDIR"/zkEnv.sh
 fi
 
-"$JAVA" -cp "$CLASSPATH" "$JVMFLAGS" \
+"$JAVA" -cp "$CLASSPATH" $JVMFLAGS \
      org.apache.zookeeper.server.SnapshotRecursiveSummary "$@"
 
 
